guides/v2.3/graphql/reference/wishlist.md

```diff
@@ -0,0 +1,94 @@
+---
+group: graphql
+title: Wish list endpoint
+---
+
+The Wish list endpoint returns the contents of a customer's wish list.
+
+### Wish list attributes
+
+Attribute | Data type | Description
+--- | --- | ---
+`items` | [Wishlistitem](#wishlistitem) | An array of items in the customer's wish list
+`items_count` | Int | The number of items in the wish list
+`name` | String | When multiple wish lists are enabled, the name the customer assigns to the wish list
+`sharing_code` | String | An encrypted code that Magento uses to link to the wish list
+`updated_at` | String | The time of the last modification to the wish list
+
+### Wish list item attributes {#wishlistitem}
+
+Attribute | Data type | Description
+--- | --- | ---
+`added_at` | String | The time when the customer added the item to the wish list
+`description` | String | The customer's comment about this item
+`id` | Int | The wish list item ID
+`product` | <a href="{{ page.baseurl }}/graphql/reference/products.html#ProductInterface">ProductInterface</a> | The ProductInterface contains attributes that are common to all types of products. Note that descriptions may not be available for custom and EAV attributes
+`qty` | Float | The quantity of this wish list item
+
+## Syntax
+
+`wishlist: WishlistOutput`
+
+## Example usage
+
+The following query returns the customer's wish list:
+
+**Request** 
+
+```
+{
+  wishlist {
+    items_count
+    name
+    sharing_code
+    updated_at
+    items {
+      id
+      qty
+      description
+      added_at
+      product {
+        sku
+        name
+      }
+    }
+  }
+}
+```
+
+**Response**
+
+```json
+{
+  "data": {
+    "wishlist": {
+      "items_count": 2,
+      "name": "Wish List",
+      "sharing_code": "KAXDj0HlM7Y2s58mllsVhSJvRj4fWIZj",
+      "updated_at": "2019-02-13 22:47:45",
+      "items": [
+        {
+          "id": 1,
+          "qty": 1,
+          "description": null,
+          "added_at": "2019-02-20 14:38:02",
+          "product": {
+            "sku": "MJ09",
+            "name": "Taurus Elements Shell"
+          }
+        },
+        {
+          "id": 2,
+          "qty": 1,
+          "description": null,
+          "added_at": "2019-02-20 14:38:28",
+          "product": {
+            "sku": "MSH11",
+            "name": "Arcadio Gym Short"
+          }
+        }
+      ]
+    }
+  }
+}
+```
```

Comment:
Create a "Mutatations" H2 and say they'll be added in a future release.

Reply:
done